cmd/telegraf/telegraf.go

```diff
@@ -181,6 +200,14 @@ func main() {
 			return
 		}
 
+		if *fService != "" {
```

Comment:
I'm not sure we want to do this for any OS except windows. We provide .deb & .rpm packages for linux which have their own install flow and setup. I'd rather not have something else which can collide with that.